pages/guides/agents/intermediate/local-agent-inspector.mdx

```diff
@@ -0,0 +1,85 @@
+import { Callout } from 'nextra/components'
+
+# Local Agent Inspector
+
+The Local Agent Inspector is a developer tool on Agentverse that enhances the debugging and monitoring experience for [Agents running on a local infrastructure](https://fetch.ai/docs/guides/agents/intermediate/agent-types#local-agents). It allows you to track and understand the real-time behavior of these Agents, whether they are connected to Agentverse or not. The tool offers developers detailed insights into your local agent’s performance and interactions, even though the agent itself is run within your infrastructure.
+
+    <Callout type="info" emoji="ℹ️">
+        Local Agents can communicate and interact with other Agents and Functions connected to the Agentverse and Fetch Network through a Mailbox. For more information on Mailbox and how to connect local Agents and their function to the Agentverse, have a look at these guides:
+        
+            - [Agent Mailbox](https://fetch.ai/docs/guides/agents/intermediate/mailbox).  
+            - [Options for running local Agents]()
+            - [Agent Functions ](https://fetch.ai/docs/guides/agents/intermediate/agent-functions). 
+            - [Agentverse Functions: register your Agents Functions on the Agentverse!](https://fetch.ai/docs/guides/agentverse/agentverse-functions/registering-agent-services). 
+    </Callout>
+
+The Inspector tool provides developers with multiple key insights, including the agent’s address, details about its local and external endpoints, the types of messages sent and received, and comprehensive data about the sender and recipient of each message. This functionality makes it easier to monitor communication flows, troubleshoot issues, and analyze performance in real-time.
+
+By visualizing this information in the Agentverse environment, the Local Agent Inspector allows you to maintain real-time oversight of how your Agent interacts with other Agents and systems, facilitating easier troubleshooting, performance analysis, and optimizing communication flows. This functionality is particularly valuable for complex applications where local agents are responsible for processing real-time tasks, interacting with other agents, or managing resources directly. It provides a way to see how your agent is performing within a networked environment, while still retaining the benefits of local execution and control.
+
+All messages data regarding a particular local agent are then presented as a list through which it is also possible to access the payload for the message being examined.
+
+If you are managing multiple local agents, for example, through a `Bureau`, the Inspector will display a complete list of all agents running locally, along with the relevant details mentioned earlier.
```

Comment:
Please update this to say that while agents running in a bureau are not supported in the initial version, we will be adding support for this very soon.

Reply:
Done